### PR TITLE
Fix/3261/Changelog Dialog

### PR DESCRIPTION
**Pull request by Hall-Ma**
# Fix disappearance of changelog entries in dialog

Close: #3261

## Description

  - ensures that new lines are recognized when the change log is read in and that a new array entry is added per line.
  - to ensure that the changelog dialogue renders content, an e2e test was created

### CODE REVIEW

File: visualization/app/codeCharta/ui/dialogs/changelogDialog/changelogDialog.component.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ export class ChangelogDialogComponent {
 	changes: Record<string, string>
 
 	constructor(@Inject(MAT_DIALOG_DATA) public data: { previousVersion: string; currentVersion: string }) {
-		let changelogLines = markdownFile.split("\n")
+		const newLineStartPattern = /(?<=>)\s+(?=<)/g
```
Comment (ce-bo) [inline visualization/app/codeCharta/ui/dialogs/changelogDialog/changelogDialog.component.ts:14]: I am not certain if the regex is correct. I checked it by regex101. Can you please confirm this?

**Review by ce-bo — APPROVED** [review]

File: visualization/app/codeCharta/ui/dialogs/changelogDialog/changelogDialog.component.ts (hunk 1 above)
Comment (BridgeAR) [reply visualization/app/codeCharta/ui/dialogs/changelogDialog/changelogDialog.component.ts:14]: We should indeed split the lines similar to before. I guess we might also remove potential carriage returns.

File: visualization/package.json [hunk 2]
```diff
@@ -128,8 +128,8 @@
 		"lint-staged": "^13.0.0",
 		"nw-builder": "^3.7.4",
 		"puppeteer": "^17.0.0",
-		"remark-html": "^13.0.1",
-		"remark-loader": "^4.0.0",
+		"raw-loader": "^4.0.2",
```
Comment (ce-bo) [inline visualization/package.json:131]: Raw loader seems to be deprecated soon.
"DEPREACTED for v5: please consider migrating to [asset modules](https://webpack.js.org/guides/asset-modules/)."

Comment (Hall-Ma) [reply visualization/package.json:131]: I replace it.

**Pull request merged**